src/lib/response-builder.js

```diff
@@ -10,7 +10,7 @@ const responseConfigProps = [
 
 module.exports = class ResponseBuilder {
 	constructor (url, responseConfig, fetchMock) {
-		this.url = url;
+		this.url = url instanceof Request ? url.url : url;
```

Comment:
rather than `Request` use `fetchMock.config.Request` as this is resilient to people using custom `fetch` implementations